treeherder/seta/analyze_failures.py

```diff
@@ -0,0 +1,142 @@
+import datetime
+import json
+import logging
+import os
+
+import requests
+from redo import retry
+
+from treeherder.seta.models import JobPriority
+from treeherder.seta.seta import get_high_value_jobs
+from treeherder.seta.update_job_priority import ManageJobPriorityTable
+
+
+LOG = logging.getLogger(__name__)
+DAYS_TO_ANALYZE = 90  # Number of days to go back and gather failures
+
+
+class AnalyzeFailures:
+    def __init__(self, **options):
+        self.start_date = (datetime.datetime.now() -
+                           datetime.timedelta(days=DAYS_TO_ANALYZE)).strftime("%Y-%m-%d")
```

Comment:
I don't know the implications of using one over the other. One is python's standard library and the other from django.
I'm only using django's timezone for the tests (since I saw it in other tests and tutorials).

@wlach do you have any preference?